torch/testing/_internal/jit_utils.py

```diff
@@ -686,3 +686,8 @@ def warmup_backward(f, *args):
             f.backward(retain_graph=True)
 
     return results
+
+# TODO: Remove me once https://bugs.python.org/issue42666 is resolved
+def make_global(*args):
+    for arg in args:
+        setattr(sys.modules[arg.__module__], arg.__name__, arg)
```

Comment:
Just OOC, why not setting it in `global()`? to avoid name conflicts?

Reply:
`global()` would make visible in `torch.testing._internal.jit_utils` module rather than in `__main__`
I believe it is a small bug in the resolver that I need to look into.